app/Config/core.php

```diff
@@ -104,36 +105,36 @@
  * included primarily as a development convenience - and
  * thus not recommended for production applications.
  */
-	//Configure::write('App.baseUrl', env('SCRIPT_NAME'));
+    //Configure::write('App.baseUrl', env('SCRIPT_NAME'));
```

Comment:
__[staging][code_sniffer]__
* Comment closer must be on a new line
* No space found before comment text; expected "// Configure::write('App.baseUrl', env('SCRIPT_NAME'));" but found "//Configure::write('App.baseUrl', env('SCRIPT_NAME'));"
* Inline comments must end in full-stops, exclamation marks, or question marks
* There must be no blank line following an inline comment